src/main/resources/static/js/toggle-view.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Created by Maaike Visser on 13-6-2017.
+ */
+
+$(function () {
+    "use strict";
+
+    var view = true;
```

Comment:
Instead of a boolean, I think it is easier to use [`toggle()`](http://api.jquery.com/toggle/)